2024/02/08 13:32:30 [error] 22#22: *1 connect() failed (113: Host is unreachable) while connecting to upstream, client: 192.168.65.1, server: api.local, request: "OPTIONS /broadcasting/auth HTTP/1.1", upstream: "fastcgi://192.168.112.5:9000", host: "api.local:8000", referrer: "http://chat.local:8081/"
2024/02/08 13:32:30 [error] 24#24: *3 connect() failed (113: Host is unreachable) while connecting to upstream, client: 192.168.65.1, server: api.local, request: "OPTIONS /broadcasting/auth HTTP/1.1", upstream: "fastcgi://192.168.112.5:9000", host: "api.local:8000", referrer: "http://chat.local:8081/"
2024/02/08 13:50:52 [error] 21#21: *5 connect() failed (113: Host is unreachable) while connecting to upstream, client: 192.168.65.1, server: api.local, request: "OPTIONS /broadcasting/auth HTTP/1.1", upstream: "fastcgi://192.168.112.5:9000", host: "api.local:8000", referrer: "http://chat.local:8081/"
2024/02/08 13:50:52 [error] 23#23: *7 connect() failed (113: Host is unreachable) while connecting to upstream, client: 192.168.65.1, server: api.local, request: "OPTIONS /broadcasting/auth HTTP/1.1", upstream: "fastcgi://192.168.112.5:9000", host: "api.local:8000", referrer: "http://chat.local:8081/"
2024/02/08 13:58:25 [error] 22#22: *1 connect() failed (113: Host is unreachable) while connecting to upstream, client: 192.168.65.1, server: api.local, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.128.7:9000", host: "api.local:8000"
2024/02/08 13:59:10 [error] 23#23: *4 connect() failed (113: Host is unreachable) while connecting to upstream, client: 192.168.65.1, server: api.local, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.128.7:9000", host: "localhost:8000"
2024/02/08 13:59:13 [error] 23#23: *4 connect() failed (113: Host is unreachable) while connecting to upstream, client: 192.168.65.1, server: api.local, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.128.7:9000", host: "localhost:8000"
